semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -1344,13 +1345,25 @@ and nullable_type (env : env) ((v1, v2) : CST.nullable_type) =
     )
   in
   let v2 = List.map (token env) (* "?" *) v2 in
-  todo env (v1, v2)
+  (match v2 with
+   | hd::tl -> TyQuestion (v1, hd)
+   | _ ->
```

Comment:
in grammar.js they use repeat1($._quest)
and _quest: $ => "?"
which means there is at least one "?", so you can write the pattern:
| [] -> raise Impossible (* see repeat1($._quest) in grammar.js *)